targets/targets.json

```diff
@@ -7428,6 +7428,12 @@
             "IAR",
             "ARMC6"
         ],
+        "components": [
+            "FPGA_CI_TEST_SHIELD"
+        ],
+        "overrides": {
+            "default-form-factor": "samsung"
```

Comment:
Can you provide more information about the form factor?
Is it documented somewhere?

Reply:
> Tested from factor is defined by MBED_CONF_TARGET_DEFAULT_FORM_FACTOR  
"default-form-factor" default value is null.
When "default-form-factor" is not set, ARDUINO form factor is used.

The pin of our board does not conform to the audio form factor, so we have adjusted it ourselves. 
Please refer to the specific pin definition in `mbed-os/targets/TARGET_Samsung/TARGET_SIDK_S5JS100/pinmap.c` and `mbed-os/targets/TARGET_Samsung/TARGET_SIDK_S5JS100/PinNames.h`

Reply:
Thanks @zhizhezhu ! I suggest to add a few comments in the PinNames.h as indicated.